ports/esp32/modnetwork_globals.h

```diff
@@ -31,6 +31,7 @@
 { MP_ROM_QSTR(MP_QSTR_AUTH_OWE), MP_ROM_INT(WIFI_AUTH_OWE) },
 #if ESP_IDF_VERSION > ESP_IDF_VERSION_VAL(5, 1, 1)
 { MP_ROM_QSTR(MP_QSTR_AUTH_WPA3_ENT_192), MP_ROM_INT(WIFI_AUTH_WPA3_ENT_192) },
+{ MP_ROM_QSTR(MP_QSTR_AUTH_WPA3_EXT_PSK), MP_ROM_INT(WIFI_AUTH_WPA3_EXT_PSK) },
```

Comment:
This constant was added in IDF 5.2.0, so a new `#if` guard needs to be added, eg:
```c
#if ESP_IDF_VERSION >= ESP_IDF_VERSION_VAL(5, 2, 0)
{ MP_ROM_QSTR(MP_QSTR_AUTH_WPA3_EXT_PSK), MP_ROM_INT(WIFI_AUTH_WPA3_EXT_PSK) },
#endif
```
And the logic in `network_common.c` changed appropriately.